app/containers/AccountProfileSetting/AccountProfileConfirm.js

```diff
@@ -1,5 +1,6 @@
 import { connect } from 'react-redux';
 import { getValues } from 'redux-form';
+import { reset } from 'redux-form';
```

Comment:
you can put it on the above live `import { getValues, reset } from 'redux-form';`